### ISSUE TITLE
The signal quality decreases after the speaker is enabled

### ISSUE THREAD

**Issue by 15021585823** [BelledonneCommunications/linphone-android#1370]
With the latest version of SDK, call SIP account after logging in. After the call is connected, the signal quality decreases after the speaker is enabled, and the voice of the other party is not clear, which will last about 20 seconds.

I've tested a dozen models, and so far only the Redmi Note8 Pro has this problem.
![17965EC4-0EE7-4a4b-9B73-3CF1A815DC8F](https://user-images.githubusercontent.com/15831485/115541674-007dac00-a2d2-11eb-9c32-4c913229ca3d.png)

**Comment by Viish** [comment]
Most probably the echo canceller on this device is crappy.
Can you attach full logs?

**Comment by 15021585823** [comment]
> Most probably the echo canceller on this device is crappy.
> Can you attach full logs?


Please check the log

> Most probably the echo canceller on this device is crappy.
> Can you attach full logs?

**Comment by 15021585823** [comment]
> Most probably the echo canceller on this device is crappy.
> Can you attach full logs?


[callLogs.txt](https://github.com/BelledonneCommunications/linphone-android/files/6354918/callLogs.txt)

Please check the log

**Comment by Viish** [comment]
Can you try to add the following code in your app after the Core is created and before it is started:
`core.mediastreamerFactory.setDeviceInfo("Xiaomi", "Redmi Note 8 Pro", "mt6785", org.linphone.mediastream.Factory.DEVICE_HAS_BUILTIN_AEC_CRAPPY, 0 , 250)`
Then try to reproduce the issue.

**Comment by 15021585823** [comment]
> Can you try to add the following code in your app after the Core is created and before it is started:
> `core.mediastreamerFactory.setDeviceInfo("Xiaomi", "Redmi Note 8 Pro", "mt6785", org.linphone.mediastream.Factory.DEVICE_HAS_BUILTIN_AEC_CRAPPY, 0 , 250)`
> Then try to reproduce the issue.

Hi, still not available, please check the log
[linphone_logs.txt](https://github.com/BelledonneCommunications/linphone-android/files/6362670/linphone_logs.txt)
![code1](https://user-images.githubusercontent.com/15831485/115816777-0edfdb00-a42c-11eb-8ae5-9cf4c8b2f036.png)

**Comment by Viish** [comment]
Indeed it's not that.
Can you try to use AAudio instead of OpenSLES? Just build the SDK with -DENABLE_AAUDIO=ON.

**Comment by 15021585823** [comment]
> Indeed it's not that.
> Can you try to use AAudio instead of OpenSLES? Just build the SDK with -DENABLE_AAUDIO=ON.

Sorry, I tried to build it, but it failed.Can you try something else, or I can send you my test code

**Comment by Viish** [comment]
Why did it fail ?

**Comment by Viish** [comment]
Yes it is a possibility, but you don't have to use docker if you have the build tools installed on your computer.

**Comment by 15021585823** [comment]
> Yes it is a possibility, but you don't have to use docker if you have the build tools installed on your computer.

 cmake --build  return error
PS D:\workspace\test\linphone-sdk> cmake --build .
Error: could not load cache
